libunistring failures with gcc-13

Error logs below.  Looks like it is not linking to the correct threads?

Presumably this should be a configure argument.  

FWIW, the build script does not have libunistring special cased so it uses the generic build:
https://github.com/StrawberryPerl/build-extlibs/blob/eb611d53149a115e0f04a40e266fcecd3d013ca4/build.sh#L1688-L1698


Version 0.9.10:
```
libtool: link: x86_64-w64-mingw32-gcc -shared .libs\\libunistring.la.lnkscript  -L/z/extlib/_bgcc13__/lib -liconv  -O2 -
mms-bitfields -Wl,--export-all-symbols -Wl,--disable-auto-import   -o .libs/libunistring-2__.dll -Wl,--enable-auto-image
-base -Xlinker --out-implib -Xlinker .libs/libunistring.dll.a
C:/spbuild/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: .libs\\localename
.o:localename.c:(.rdata$.refptr.pthread_mutexattr_gettype[.refptr.pthread_mutexattr_gettype]+0x0): undefined reference t
o `pthread_mutexattr_gettype'
collect2.exe: error: ld returned 1 exit status
make[3]: *** [Makefile:4215: libunistring.la] Error 1
make[3]: Leaving directory '/z/extlib/_bgcc13__.src/libunistring-0.9.10/lib'
make[2]: *** [Makefile:3581: all] Error 2
make[2]: Leaving directory '/z/extlib/_bgcc13__.src/libunistring-0.9.10/lib'
make[1]: *** [Makefile:1559: all-recursive] Error 1
make[1]: Leaving directory '/z/extlib/_bgcc13__.src/libunistring-0.9.10'
make: *** [Makefile:1487: all] Error 2
```

Version 1.1:
```
libtool: link: rm -fr  .libs/libunistring.dll.a .libs/libunistring.la.lnkscript
libtool: link: x86_64-w64-mingw32-gcc -shared .libs\\libunistring.la.lnkscript  -L/z/extlib/_bgcc13__/lib -liconv  -O2 -
mms-bitfields -Wl,--export-all-symbols   -o .libs/libunistring-5__.dll -Wl,--enable-auto-image-base -Xlinker --out-impli
b -Xlinker .libs/libunistring.dll.a
C:/spbuild/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: .libs\\libunistri
ng_la-localename.o:localename.c:(.rdata$.refptr.pthread_mutexattr_gettype[.refptr.pthread_mutexattr_gettype]+0x0): undef
ined reference to `pthread_mutexattr_gettype'
collect2.exe: error: ld returned 1 exit status
make[2]: *** [Makefile:5285: libunistring.la] Error 1
make[2]: Leaving directory '/z/extlib/_bgcc13__.src/libunistring-1.1/lib'
make[1]: *** [Makefile:7273: install] Error 2
make[1]: Leaving directory '/z/extlib/_bgcc13__.src/libunistring-1.1/lib'
make: *** [Makefile:2018: install-recursive] Error 1
```

--- a/5034
+++ b/5034
@@ -56,7 +56,7 @@ t1lib-5.1.2
 libffi-3.2.1
 libuv-v1.28.0
 
-libunistring-0.9.10
+libunistring-1.1
 libidn2-2.1.1
 
 ###### PDL related

--- a/build.sh
+++ b/build.sh
@@ -327,6 +327,19 @@ xxrun make install
 ;;
 
 # ----------------------------------------------------------------------------
+libunistring-*)
+cd $WRKDIR/$PACK
+save_configure_help
+xxrun ./configure $HOSTBUILD --prefix=$OUT --disable-dependency-tracking \
+                             --enable-static=no --enable-shared=yes \
+                             --enable-threads=windows \
+                             CFLAGS=" -O2 -I$OUTINC -mms-bitfields" LDFLAGS="-L$OUTLIB"
+patch_libtool
+xxrun make
+xxrun make install
+;;
+
+# ----------------------------------------------------------------------------
 libxml2-*)
 cd $WRKDIR/$PACK
 save_configure_help

--- a/sources.list
+++ b/sources.list
@@ -371,6 +371,7 @@ http://ftp.gnu.org/gnu/libidn/libidn2-2.1.1a.tar.gz -O libidn2-2.1.1.tar.gz
 
 #libunistring is required by libidn2
 https://ftp.gnu.org/gnu/libunistring/libunistring-0.9.10.tar.gz
+https://ftp.gnu.org/gnu/libunistring/libunistring-1.1.tar.gz
 
 #URL http://curl.haxx.se/download/
 http://curl.haxx.se/download/curl-7.45.0.tar.gz

